react-player/build.boot

```diff
@@ -0,0 +1,30 @@
+(def +lib-version+ "1.5.1")
+(def +version+ (str +lib-version+ "-0"))
+
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.9.0" :scope "test"]
+                  [cljsjs/react "16.0.0-0"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(task-options!
+ pom  {:project     'cljsjs/react-player
+       :version     +version+
+       :description "A React component for playing a variety of URLs, including file paths, YouTube, Facebook, Twitch, SoundCloud, Streamable, Vimeo, Wistia and DailyMotion"
+       :url         "https://github.com/CookPete/react-player"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+    (download :url (str "https://unpkg.com/react-player@1.5.1/dist/ReactPlayer.js"))
```

Comment:
Please use versioned url to download the archive or the files. Use the `+lib-version+` var you have defined to build the url:

```clj
(def +lib-version+ "1.2.3")

...

(download :url (format "https://github.com/example/project/archive/%s.zip" +lib-version+)
          :unzip true)
```

Reply:
Fixed.